theme/typography.ts

```diff
@@ -90,12 +90,17 @@ export const textStyles = defineTextStyles({
   },
   'caption.02': {
     description: 'caption.02',
-    value: { ...commonDiatypeStyles, fontSize: '0.8125rem', lineHeight: '1.25rem' },
+    value: { ...commonDiatypeStyles, fontSize: '0.8125rem', lineHeight: '1rem' },
   },
 
   'mono.01': {
     description: 'mono.01',
-    value: { fontFamily: firaCode, fontSize: '1.0625rem', lineHeight: '1.75rem' },
+    value: {
+      fontFamily: firaCode,
+      fontSize: '1rem',
+      lineHeight: '1.5rem',
+      letterSpacing: '.08rem',
+    },
```

Comment:
@mica000 @fabric-8 are these changes consistent with design system?

Reply:
I did check this in the designs, that is why I made the change here.

Reply:
Image of change for ref, I did check the review page designs so let me know if not accurate...
 
<img width="525" alt="Screenshot 2023-11-24 at 12 23 16 PM" src="https://github.com/leather-wallet/extension/assets/6493321/cb69e3f6-45d3-402c-8073-acde4e0d0d12">